samples/README.md

```diff
@@ -0,0 +1,21 @@
+## Overview
```

Comment:
We can create different folders under samples so as to allow addition of other samples going forward.
Also this should be linked from the main README. Something like for sample examples look at the samples folder.

Reply:
Created a subfolder in samples and added link to sample folder in main README